vector/src/main/java/im/vector/app/features/location/live/map/LiveLocationMapViewFragment.kt

```diff
@@ -141,6 +147,13 @@ class LiveLocationMapViewFragment @Inject constructor() : VectorBaseFragment<Fra
         }
     }
 
+    private fun listenMapLoadingError(mapView: MapView) {
+        this.mapView = mapView
```

Comment:
I think it would make more sense to do it in setupMap

Reply:
It is actually called in `setupMap()`: `(mapFragment.view as? MapView)?.let(::listenMapLoadingError)` but it can only be called inside the callback of `getMapAsync` to have access to the `MapView`. What was you suggestion?

Reply:
Just to move it outside this function, I think it is more logical to store the map reference in the `setupMap` method instead of when listening to the map loading errors. (Even if it will not change the behaviour)

Reply:
Okay I see, thanks. I agree I will update it.

Reply:
Updated see [3f7f724](https://github.com/vector-im/element-android/pull/6712/commits/3f7f72413815e5f8137641727ad1dc73afa0949c)